lib/Textile/Models/index.ts

```diff
@@ -4,9 +4,16 @@ export interface TextileConfig {
   RELEASE_TYPE?: string
   TEXTILE_CAFE_GATEWAY_URL?: string
   TEXTILE_CAFE_OVERRIDE?: string
+  TEXTILE_CAFE_TOKEN?: string
   SELF_MANAGE_APP_STATE?: boolean
```

Comment:
Those 3 cafe fields can be removed above

Reply:
Right, nice catch. Gone